packages/framework/undo-redo/package.json

```diff
@@ -92,6 +92,8 @@
   },
   "typeValidation": {
     "version": "0.59.3000",
-    "broken": {}
+    "broken": {
+      "ClassDeclaration_SharedSegmentSequenceRevertible": {"forwardCompat": false}
+    }
```

Comment:
This was the only type test change.

Reply:
@Abe27342 @anthony-murphy Can you help me find the source of this change? This is on the 1.2 release branch; we've not been running type tests properly as part of the releases, so this may have been in an earlier release.